### PR TITLE
Add classes and create flex file

### PR DESCRIPTION
**Pull request by flora8984461**
Signed-off-by: Yi Liu <yi.liu@eclipse-foundation.org>

### CODE REVIEW

File: less/_components/classes-flex.less [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+*/
+
+// Classes styles: Flex
+// css classes for Flex layout.
+
+.display-flex {
+  display: flex;
+}
+
+.vertical-align {
```
Comment (autumnfound) [inline less/_components/classes-flex.less:21]: this should probably be left in grids less. While this has flex, it isn't a flex feature and is more using it as a part of the solution.

File: less/_components/classes-flex.less [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  .flex-grow {
+    flex-grow: 1;
+  }
+}
+
+.flex-wrap {
+  flex-wrap: wrap;
+}
+
+@media (min-width: @screen-sm-min) {
```
Comment (autumnfound) [inline less/_components/classes-flex.less:46]: the justify content classes should be left in grid as they have nothing to do with flex

**Review by autumnfound — CHANGES_REQUESTED** [review]

File: less/_components/classes-flex.less (hunk 1 above)
Comment (flora8984461) [reply less/_components/classes-flex.less]: moved back

File: less/_components/classes-flex.less (hunk 2 above)
Comment (flora8984461) [reply less/_components/classes-flex.less]: moved back

**Pull request merged**